workbench-core/example/app/src/configs/staticRouteConfig.ts

```diff
@@ -199,15 +199,23 @@ export const routesMap: RoutesMap = {
       }
     ]
   },
-  [`/authorization/groups/${groupIDRegExpAsString}/getUsers`]: {
+  [`/authorization/groups/${groupIDRegExpAsString}/get-users`]: {
```

Comment:
Do we need the `get-users` here or can this be just a `GET` on the groupId?

Reply:
we ran into an issue with the `groupId` since it can be any non-whitespace character between 1 and 128 characters. If we dont have something at the end of the route (ie `/get-users` here) the incoming route isnt matched correctly. @insignias made a suggestion of restricting the `groupId` further by not allowing `/` so that `/` can be used as a delimiter. We werent sure of best practices so we left it as is

Reply:
+1